src/plugin.js

```diff
@@ -403,6 +355,59 @@ const eme = function(options = {}) {
         }
       }
     },
+    initLegacyFairplay() {
+      const playerOptions = getOptions(player);
+      const handleFn = (event) => {
+        // TODO convert to videojs.log.debug and add back in
+        // https://github.com/videojs/video.js/pull/4780
+        // videojs.log('eme', 'Received a \'webkitneedkey\' event');
```

Comment:
should it be removed?

Reply:
Yeah I'm thinking maybe we just remove the whole comment as this pertains only to legacy fairplay. Good catch.

Reply:
or uncommented and changed to `videojs.log.debug()`?

Reply:
I am definitely for extra logging. I wonder why it was not included before :)

Reply:
added `log.debug` for each event that was missing it and removed the TODOs.